NEWS.md

```diff
@@ -1,5 +1,10 @@
 # covr (development version)
 
+* Added `covr.record_tests` option. When `TRUE`, this enables the recording of
+  the trace of the tests being executed and adds an itemization of which tests
+  result in the exeuction of each trace. For more details see
+  `?covr.record_tests`
```

Comment:
Can you add `(@yourname, #issuenumber)` to this news bullet? And also add yourself to the `Authors@R` section of the package (just your name, no need for an email)

Reply:
Thanks - Added in 92d944d83e7b91f5044ceb779b4f1b8b3a4b7ea9, d96227e4d55c063b7fc13d874f855f563e586ad8